### PR TITLE
configuration file to replace hard coded gerrit host name

### PR DESCRIPTION
**Pull request by lmilbaum**

### CODE REVIEW

**Comment by lmilbaum** [comment]
I will be trying to downgrade the python version used on my mac, to align with the one already used.

**Comment by lmilbaum** [comment]
3.6 is no longer supported by brew. 3.7 is the supported python version

File: review_wof/__init__.py [hunk 1]
```diff
@@ -1,19 +1,20 @@
 """review_wof.py - Build report of patches and reviews
 """
+import yaml
 from flask import Flask, render_template
 
+from .model import patch_reviewers_view, patch_summary_view, patches_view
```
Comment (ifireball) [inline review_wof/__init__.py:5]: I prefer it the way it was, and this is not related to the change you're proposing, so leave it like it is

File: review_wof/__init__.py [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-from .model import patch_reviewers_view
-
 
 application = Flask(__name__)
 
 
 @application.route('/')
 def index():
-    patches = GerritPatches()
+    with open("config.yml", 'r') as ymlfile:
```
Comment (ifireball) [inline review_wof/__init__.py]: I think an environment variable would be a more forward-looking way to store configuration (5 factor, etc.), also all this code belongs in the model layer, not here in the view layer

If we want to have something the represent a list of servers - we need a GerritServers class in the model layer, but I actually do not think that is a useful abstraction because dividing patches by server is only marginally useful, as teams can typically have work that spans multiple servers. What we should aim for instead, is to merge together information from multiple servers and in the long term - allow division by team from the UI.

File: review_wof/model/gerrit_patches.py [hunk 3]
```diff
@@ -18,14 +16,18 @@
 
 
 class GerritPatches(GeneratorDSC):
-    def __init__(self, since=None, until=None):
+    def __init__(self, host='gerrit.ovirt.org', since=None, until=None):
```
Comment (ifireball) [inline review_wof/model/gerrit_patches.py]: I actually do not think GerritPatches should be a single-server - based class, instead I think we need to expand it internally to be able to collect information from multiple servers.

File: Pipfile [hunk 4]
```diff
@@ -13,4 +13,4 @@ pytest = "*"
 review-wof = {editable = true,path = "."}
 
 [requires]
-python_version = "3.6"
+python_version = "3.7"
```
Comment (ifireball) [inline Pipfile:16]: I'd like to leave the current version as-is, 3.7 is not conveniently available on RHEL7, while 3.6 is in EPEL.

**Review by ifireball — CHANGES_REQUESTED** [review]

**Comment by ifireball** [comment]
> 3.6 is no longer supported by brew. 3.7 is the supported python version

I don`t see what brew has to do with any of this - also it does not make sense given that [3.6 was the version RHEL8 was release with](https://developers.redhat.com/blog/2018/11/14/python-in-rhel-8/), it should be supported in the foreseeable future

You can try installing [pyenv](https://github.com/pyenv/pyenv) to help you manage Python versions on your development machine, the [pipenv docs claim to have a good integration with it](https://pipenv.kennethreitz.org/en/latest/advanced/#automatic-python-installation).

File: review_wof/__init__.py (hunk 1 above)
Comment (lmilbaum) [reply review_wof/__init__.py:5]: isort?

File: review_wof/__init__.py (hunk 2 above)
Comment (lmilbaum) [reply review_wof/__init__.py]: I need to read your comment few times to understand what you are aiming for :-)

File: review_wof/__init__.py (hunk 1 above)
Comment (ifireball) [reply review_wof/__init__.py:5]: well, I don't want to decide on a particular style at this point, so you might as well disable it in your IDE for this project.

in any case you should not mix unrelated re-factoring and feature additions in the same PR.

if you do want to use isort - go ahead and run it on all the files and submit a PR with that, and perhaps also some CI action to verify it on new PRs.

Comment (lmilbaum) [reply review_wof/__init__.py:5]: ack

File: review_wof/__init__.py (hunk 2 above)
Comment (lmilbaum) [reply review_wof/__init__.py]: Have done that several times with no avail

File: review_wof/model/gerrit_patches.py (hunk 3 above)
Comment (lmilbaum) [reply review_wof/model/gerrit_patches.py]: Agree. Then, do we want to remove the default value?

File: Pipfile (hunk 4 above)
Comment (lmilbaum) [reply Pipfile:16]: I am working on my home laptop which is MAC. Can we find a middle ground which works for both of us?

**Comment by lmilbaum** [comment]
there is another program used in OS-X, named brew.  I have tried using pyenv and successed massing up my laptop. It doesn't play nice with pipenv.

File: review_wof/__init__.py (hunk 2 above)
Comment (lmilbaum) [reply review_wof/__init__.py]: Can I assume a DataSource is a Git server. Valid values would be: Gerrit, GitHub, Gitlab.
